app/models/invenio_rdm_record_converter.rb

```diff
@@ -51,10 +52,11 @@ def to_json(options={})
 
   private
 
-  def filename_and_content_path
+  def file_info
     {
       "filename": @generic_file.filename,
-      "content_path": generic_file_content_path(@generic_file.content.checksum.value)
+      "content_path": generic_file_content_path(@generic_file.content.checksum.value),
+      "original_checksum": @generic_file.original_checksum
```

Comment:
From what I checked, `original_checksum` matched the file's md5 checksum. 🤷‍♂️

Reply:
wow ok, they generate a SHA1 and an MD5 from every file then...

Reply:
@fenekku seems so, I wonder what else there could be...